man/kbmadm.8

```diff
@@ -123,7 +123,7 @@ Any option listed in the
 .Nm zpool
 .Cm create
 subcommand in
-.Xr zpool
```

Comment:
Is this correct?

Reply:
Yes.  Wanna mention the section (zpool(8)).